src/config/config.ts

```diff
@@ -94,6 +94,17 @@ function applyDefaultConfigurationValues(serverConfig: any) {
             }
         }
     });
+
+    // This pass handles boolean values that aren't in ServerConfigDefaults
+    // Without this, these boolean properties will be rendered as string
+    Object.keys(serverConfig).forEach(key => {
+        if (typeof serverConfig[key] === 'string') {
+            const lowerVal = serverConfig[key].toLowerCase();
```

Comment:
no big deal but i like to get people to use regexp in such cases.  e.g.
```
/true|false/i.test(serverConfig[key])
```
now you don't have to mess around with lowercasing and you don't have to do null/undefined check.

Reply:
Done